runtime/gc_include/j9modron.h

```diff
@@ -192,16 +192,16 @@ typedef jvmtiIterationControl J9MODRON_REFERENCE_CHAIN_WALKER_CALLBACK(J9Object
 #define J9GC_HASH_SALT_COUNT_STANDARD 1
 #define J9GC_HASH_SALT_NURSERY_INDEX 0
 
-#define J9_GC_MANAGEMENT_POOL_JAVAHEAP	 				0x1
-#define J9_GC_MANAGEMENT_POOL_TENURED 				0x2
-#define J9_GC_MANAGEMENT_POOL_TENURED_SOA 			0x4
-#define J9_GC_MANAGEMENT_POOL_TENURED_LOA 			0x8
-#define J9_GC_MANAGEMENT_POOL_NURSERY_ALLOCATE		0x10
-#define J9_GC_MANAGEMENT_POOL_NURSERY_SURVIVOR		0x20
-#define J9_GC_MANAGEMENT_POOL_REGION_OLD 				0x40
-#define J9_GC_MANAGEMENT_POOL_REGION_EDEN 				0x80
-#define J9_GC_MANAGEMENT_POOL_REGION_SURVIVOR 			0x100
-#define J9_GC_MANAGEMENT_POOL_REGION_RESERVED 			0x200
+#define J9_GC_MANAGEMENT_POOL_JAVAHEAP	 				0x8
+#define J9_GC_MANAGEMENT_POOL_TENURED 				0x10
+#define J9_GC_MANAGEMENT_POOL_TENURED_SOA 			0x20
+#define J9_GC_MANAGEMENT_POOL_TENURED_LOA 			0x40
+#define J9_GC_MANAGEMENT_POOL_NURSERY_ALLOCATE		0x80
+#define J9_GC_MANAGEMENT_POOL_NURSERY_SURVIVOR		0x100
+#define J9_GC_MANAGEMENT_POOL_REGION_OLD 				0x200
+#define J9_GC_MANAGEMENT_POOL_REGION_EDEN 				0x400
+#define J9_GC_MANAGEMENT_POOL_REGION_SURVIVOR 			0x800
+#define J9_GC_MANAGEMENT_POOL_REGION_RESERVED 			0x1000
```

Comment:
Please explain how changing these numbers is related to the issue at hand.

Reply:
internal NonHeap memorypool IDs are from 0x1 to 0x4, internal Heap memoryPool IDs was signed every bit from 0x1 to 0x200, the IDs 0x1, 0x2 and 0x4 are used by both Non-heap and Heap memoryPools, for external  memoryPool IDs, we add on J9VM_MANAGEMENT_POOL_HEAP flag for distinguishing heap and non-heap ids, the ID changes are not necessary, but just want to avoid the potential conflict in case we instead use internal memorypool IDs for mapping the name and the usages.